src/sagas/repository.js

```diff
@@ -136,10 +136,17 @@ export function* createDataset() {
   }
 }
 
-export function* getDatasets() {
-  // TODO: add limit and offset
+export function* getDatasets({ payload }) {
+  const offset = payload.offset || 0;
+  const limit = payload.limit || 5;
+  const filter = payload.searchString || '';
+  const sort = payload.sort || 'created_date';
+  const direction = payload.direction || 'desc';
   try {
-    const response = yield call(authGet, '/api/repository/v1/datasets');
+    const response = yield call(
+      authGet,
+      `/api/repository/v1/datasets?offset=${offset}&limit=${limit}&sort=${sort}&direction=${direction}&filter=${filter}`,
```

Comment:
Can't we pass an object with params into authGet/axios instead of doing the string interpolation ourselves here?